src/libraries/System.Runtime/tests/System/DecimalTests.cs

```diff
@@ -1500,6 +1500,14 @@ public static void ToString_InvalidFormat_ThrowsFormatException()
             Assert.Throws<FormatException>(() => f.ToString("E" + intMaxPlus1String));
         }
 
+        [Theory]
+        [InlineData("3.00")]
```

Comment:
Aren't there a bunch of other possible inputs here -- or conversely, should this just be a case added to an existing test?

Reply:
As @pgovind [said](https://github.com/dotnet/runtime/pull/48608/files#r583909909), "Yup. This is just the start." We will be creating an issue for later in the release cycle to strategically and comprehensively improve the coverage of all of the formatting/parsing logic.

Reply:
This specific scenario is the one that ASP.NET reported as broken, FYI.

Reply:
> Aren't there a bunch of other possible inputs here

There are. The plan is to have a working group some time soon to improve our unit tests for the parsing and formatting routines. I expect that this unit test will be folded into whatever we create at that point OR we'll add those inputs to this unit test.

Reply:
OK